platforms/web/vite.config.ts

```diff
@@ -114,6 +114,7 @@ export default ({ mode, command }: ConfigEnv): UserConfigExport => {
       cssCodeSplit: false,
       sourcemap: true,
       minify: true,
+      target: ['es2020', 'edge88', 'firefox78', 'chrome68', 'safari14'],
```

Comment:
If we want to differentiate between minimum browser support (e.g. a private repo), I suggest using this package: https://github.com/marcofugaro/browserslist-to-esbuild. Where we can load a different `.browserslistrc`

In the current approach i kept it simple.